### PR TITLE
lightbox gallery

### PR DESCRIPTION
**Pull request by harishkumar-srijan**
lightbox gallery

### CODE REVIEW

File: stories/Components/UIcomponents/Gallery/LightboxGallery/LightboxGallery.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    <div className="grid-x grid-margin-x lightbox-gallery-row">
+      <div className={['medium-3 cell lightbox-gallery-row__left-col'].join(' ')}>
+        <div className="lightbox-gallery-row__content">
+          <Heading type="5" label={titleText} />
+          <P label={descText} />
+          <Ctalink label={buttonText} />
+        </div>
+      </div>
+      <div className="medium-9 cell lightbox-gallery-row__right-col overflow-hidden">
+        <div>
```
Comment (shishirphadke) [inline stories/Components/UIcomponents/Gallery/LightboxGallery/LightboxGallery.jsx:25]: @harishkumar-srijan why this empty `div`?

File: stories/Components/UIcomponents/Gallery/LightboxGallery/LightboxGallery.jsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          <div className="lightbox-gallery-images">
+            <ul >
+              {data.map((item) => {
+                 return (
+                    <>
+                    <li>
+                      <a href={item.galleryimg1} class="glightbox2" data-glightbox="description: .custom-desc1; descPosition: left;">
+                          <div className="image"><img src={item.galleryimg1} alt="image" /> </div>
+                            <div class="custom-desc1 image-section__description">
+                              <figcaption class="image__caption">
```
Comment (shishirphadke) [inline stories/Components/UIcomponents/Gallery/LightboxGallery/LightboxGallery.jsx:35]: @harishkumar-srijan `figcaption` needs to be wrapped inside `figure` tag. Kindly fix the same.

File: stories/Components/UIcomponents/Gallery/LightboxGallery/lightboxgallery.scss [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+      position: absolute;
+      bottom: 0;
+      display: none;
+      .image__caption {
+        display: block;
+        border-bottom: 0;
+        padding: $spacing-32 $spacing-24;
+        margin-top: 0;
+
+        .image__caption_para {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Components/UIcomponents/Gallery/LightboxGallery/lightboxgallery.scss]: done

File: stories/Components/UIcomponents/Gallery/LightboxGallery/lightboxgallery.scss [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+          background-image: none;
+        }
+        &:hover {
+           @include devicebreak(large) {
+            .image-section__description {
+              display: block;
+            }
+          }
+        }
+        .image{
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/Components/UIcomponents/Gallery/LightboxGallery/lightboxgallery.scss]: done

File: stories/Components/UIcomponents/Gallery/LightboxGallery/LightboxGallery.jsx (hunk 2 above)
Comment (harishkumar-srijan) [reply stories/Components/UIcomponents/Gallery/LightboxGallery/LightboxGallery.jsx:35]: done

File: stories/assets/js/lib/glightbox.js [hunk 5]
```diff
@@ -0,0 +1,3696 @@
+(function (global, factory) {
```
[the remark this answers is not in the document]
Comment (harishkumar-srijan) [reply stories/assets/js/lib/glightbox.js:1]: 1. minified version used

**Pull request merged**